helm/charts/hpe-flexvolume-driver/README.md

```diff
@@ -34,7 +34,9 @@ The following table lists the configurable parameters of the HPE FlexVolume Driv
 | flexVolumeExec     | This is the path where the FlexVolume binary gets installed on the host.                             | default     |
 | storageClass.name  | The name to assign the created StorageClass.                                          | hpe-standard |
 | storageClass.create | Enables creation of StorageClass to consume this hpe-flexvolume-driver instance.                              | true        |
-| storageClass.defaultClass | Whether to set the created StorageClass as the clusters default StorageClass.                                  | false       |
+| storageClass.defaultClass | Whether to set the created StorageClass as the clusters default StorageClass.                                | false       |
+| nimble.config      | HPE Nimble Storage volume config parameters.                                                                        | -           |
+| cv.config      | HPE Cloud Volumes volume config parameters.                                                                             | -           |
```

Comment:
I think we need to document an example data structure in the readme for these.

Reply:
added example params..